### PR TITLE
Create ability_template.rst

### PR DESCRIPTION
**Pull request by v-h-gn**
* Add ability template

### CODE REVIEW

**Review by jphuse — DISMISSED** [review]
as long as there is no inline comments unless already in the config

File: source/_page_templates/ability_template.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. [insert ability name here, lowercase, one word]:
+############
+ABILITY NAME
+############
+
```
Comment (ChristopherWMM) [inline source/_page_templates/ability_template.rst:5]: Element/subelement?

File: source/_page_templates/ability_template.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Description
+###########
+
+Paste description here. Insert gif here |ABILNAMEHERE1|
```
Comment (ChristopherWMM) [inline source/_page_templates/ability_template.rst:10]: In-game image or gif should probably go above or to the side of the description, not below.

File: source/_page_templates/ability_template.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+
+Usage
+#####
+Below are the steps to use this ability
+
+Insert in-game gif here (add another substition)
```
Comment (ChristopherWMM) [inline source/_page_templates/ability_template.rst]: How does this differ from the above gif?

File: source/_page_templates/ability_template.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Description
+###########
+
+Paste description here. Insert gif here |ABILNAMEHERE1|
+
+
+Usage
+#####
+Below are the steps to use this ability
```
Comment (ChristopherWMM) [inline source/_page_templates/ability_template.rst]: Is this going to stay after info is replaced?

File: source/_page_templates/ability_template.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+Insert in-game gif here (add another substition)
+
+Function 1: [] then [] then []
```
Comment (ChristopherWMM) [inline source/_page_templates/ability_template.rst:18]: This can just be the copy-pasted usage.

Comment (ChristopherWMM) [reply source/_page_templates/ability_template.rst:18]: Maybe add in some tip sections here?

File: source/_page_templates/ability_template.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+Insert in-game gif here (add another substition)
+
+Function 2: [] then [] then []
+
+
+
+.. note:: Practice makes perfect!
```
Comment (ChristopherWMM) [inline source/_page_templates/ability_template.rst]: Idk about this being on every page.

File: source/_page_templates/ability_template.rst (hunk 2 above)
Comment (vtovmasian) [reply source/_page_templates/ability_template.rst:10]: It is on the side?

File: source/_page_templates/ability_template.rst (hunk 3 above)
Comment (vtovmasian) [reply source/_page_templates/ability_template.rst]: Its the usage gif, slowed down showing the specific steps

File: source/_page_templates/ability_template.rst (hunk 4 above)
Comment (vtovmasian) [reply source/_page_templates/ability_template.rst]: It doesn't have to

File: source/_page_templates/ability_template.rst (hunk 5 above)
Comment (vtovmasian) [reply source/_page_templates/ability_template.rst:18]: That is what the note was for

and the copy pasted Usage is often insufficient to explain certain functions like Shifting Between Ice and Water on SurgeWall, or using Combos

File: source/_page_templates/ability_template.rst (hunk 6 above)
Comment (vtovmasian) [reply source/_page_templates/ability_template.rst]: Thats the tip section lol

File: source/_page_templates/ability_template.rst [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+Function 1: [] then [] then []
+
+Insert in-game gif here (add another substition)
+
+Function 2: [] then [] then []
+
+
+
+.. note:: Insert Tips here (i.e Practice makes perfect!)
```
Comment (ChristopherWMM) [inline source/_page_templates/ability_template.rst]: Why not use the tip block for tips?

File: source/_page_templates/ability_template.rst (hunk 1 above)
Comment (ChristopherWMM) [reply source/_page_templates/ability_template.rst:5]: I still think this should be explicitly stated even though the ability will be under an elemental category.

File: source/_page_templates/ability_template.rst (hunk 3 above)
Comment (ChristopherWMM) [reply source/_page_templates/ability_template.rst]: Might be worth saying that then.

**Review by ChristopherWMM — CHANGES_REQUESTED** [review]

File: source/_page_templates/ability_template.rst (hunk 7 above)
Comment (vtovmasian) [reply source/_page_templates/ability_template.rst]: there is a tip block..

**Review by ChristopherWMM — APPROVED** [review]

**Review by jphuse — APPROVED** [review]

**Pull request merged**